Content.Client/Administration/UI/SpawnExplosion/SpawnExplosionEui.cs

```diff
@@ -0,0 +1,76 @@
+using Content.Client.Eui;
+using Content.Shared.Administration;
+using Content.Shared.Eui;
+using JetBrains.Annotations;
+using Robust.Client.Graphics;
+using Robust.Shared.Map;
+
+namespace Content.Client.Administration.UI.SpawnExplosion;
+
+[UsedImplicitly]
+public sealed class SpawnExplosionEui : BaseEui
+{
+    [Dependency] private readonly IOverlayManager _overlayManager = default!;
+
+    private readonly SpawnExplosionWindow _window;
+    private ExplosionDebugOverlay? _debugOverlay;
+
+    public SpawnExplosionEui()
+    {
+        IoCManager.InjectDependencies(this);
+        _window = new SpawnExplosionWindow(this);
+        _window.OnClose += () => SendMessage(new SpawnExplosionEuiMsg.Close());
+    }
+
+    public override void Opened()
+    {
+        base.Opened();
+        _window.OpenCentered();
+    }
+
+    public override void Closed()
+    {
+        base.Closed();
+        _window.OnClose -= () => SendMessage(new SpawnExplosionEuiMsg.Close());
```

Comment:
unsubscribing a lambda like this isnt guaranteed to work, you need to make it a member method or named delegate iirc